libnd4j/include/execution/impl/ThreadPool.cpp

```diff
@@ -78,6 +78,7 @@ ThreadPool::ThreadPool() {
   _threads.resize(_available.load());
   _interfaces.resize(_available.load());
 
+#ifndef __NEC__
```

Comment:
I think this and the other NEC related additions are by mistake as the other PRs were removing them?